README.md

```diff
@@ -117,7 +123,38 @@ The user should check the [Azure Portal](https://portal.azure.com/#home) to veri
 
 To deploy [Binderhub](https://binderhub.readthedocs.io/) to Azure use the deploy button below.
 
-[![Deploy to Azure](https://azuredeploy.net/deploybutton.svg)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Ftmbgreaves%2Fbinderhub-deploy%2Fchange-urls-to-upstream%2Fazure%2Fpaas%2Farm%2Fazure.deploy.json)
+[![Deploy to Azure](https://azuredeploy.net/deploybutton.svg)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Ftmbgreaves%2Fbinderhub-deploy%2Fmaster%2Fazure%2Fpaas%2Farm%2Fazure.deploy.json)
+
+### Monitoring deployment progress
+
+To monitor the progress of a blue-button deployment, go to the [Azure portal](https://portal.azure.com/) and select 'Resource Groups' from the left hand pane. Then in the central pane select the resource group you chose to deploy into. This will give you a right hand pane containing the resources within the group. You may need to 'refresh' until you see a new container instance. When it appears, select it, then in the new pane go to 'Settings->Containers'. You should see your new container listed. Select it, then in the lower right hand pane select 'Logs'. You may need to 'refresh' this to display the logs, possibly multiple times until the container starts up.
+
+### Retrieving deployment output from Azure
```

Comment:
We may want to add a note in this section about pulling over the ssh keys. I think they're automatically generated as `id_rsa`, so we'll want to remind people to rename them when they're copying them over so they don't overwrite they're own `id_rsa` keys.

Reply:
Good thought! I can fix that in the upload.

Reply:
Fixed in https://github.com/alan-turing-institute/binderhub-deploy/pull/22/commits/ebaf54581f86889c4a60a359022b551d93a28637